app/Http/Middleware/TrustHosts.php

```
@@ -13,8 +16,33 @@ class TrustHosts extends Middleware
+                    $url = Url::fromString($settings->fqdn);
+                    $host = $url->getHost();
+
+                    return $host ?: null;
+                }
+            } catch (\Exception $e) {
+                // If instance settings table doesn't exist yet (during installation),
+                // return null to fall back to APP_URL only
+            }
+
+            return null;
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'hosts'. (undefined)

(StaticAccess)